modules/acs-architecture-connections.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-architecture-connections_{context}"]
+= {product-title-short} connection protocols and default ports
+
+The protocols and default ports for some connections within {product-title-short} and between {product-title-short} and external components are shown in the following table. You might need this information for configuring the firewalls in your system to allow traffic on certain ports.
+
+.{product-title-short} connections between components
+
+[cols="4",options="header"]
+|===
+| Component or external entity | Connection type | Port | Additional information
+
+| Central and sensor on secured cluster
+a| * TCP/HTTPS
+* gRPC
+| 443
+| Sensor and Central primarily communicate over a bidirectional gRPC stream, initiated by sensor to Central's port 443.
+
+
+| Central and user ({product-title-short} web portal or REST API)
+| HTTPS or gRPC
+| 443
+|
+
+| Central and user (CLI)
+| gRPC
+| 443
+|
+
+| Central and vulnerability feeds
+| HTTPS
+| 443
+|
+
+| Scanner and Scanner-DB
+| TCP
+| 5432
+|
+
+| Sensor and admission controller
+| gRPC
+| 443
+| This is a bidirectional gRPC stream, initiated by the admission controller to sensor's port 443.
+
+
+| Sensor and collector
+| gRPC
+| 443
+| This connection is bidirectional and initiated by collector to Central's port 443. If node scanning is enabled for {ocp}, a connection on port 8444 is initiated by sensor to compliance (running in the collector pod).
```

Comment:
```suggestion
| Sensor and Admission Controller
| gRPC
| 443
| This is a bidirectional gRPC stream, initiated by the Admission Controller to Sensor's port 443.


| Sensor and collector
| gRPC
| 443
| This connection is bidirectional and initiated by collector to Central's port 443. If node scanning is enabled for {ocp}, a connection on port 8444 is initiated by Sensor to compliance (running in the Collector pod).
```
From my understanding, and please correct me if I'm wrong, `Admission Controller`, `Sensor` and `Collector` are all proper nouns that should be capitalized. Unfortunately searching through our repo found many entries of each both capitalized and not, I think we should make it all consistent in the future.

Reply:
Correct. For Red Hat components, and Kubernetes components, we capitalize per IBM and [Red Hat Style](https://redhat-documentation.github.io/supplementary-style-guide/#operator). We are trying to be consistent going forward.